typing_tests/pine-typed-result.ts

```diff
@@ -223,7 +223,6 @@ export let aStringOrUndefined: string | undefined;
 	aNumber = result.id;
 	aString = result.device_name;
 	aNumber = result.actor[0].id;
-	aString = result.actor[0].created_at;
```

Comment:
Yes this is a change, but the real type will always be `undefined` since it doesn't actually exist in any response..